nixos/modules/services/hardware/monado.nix

```
@@ -0,0 +1,86 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  inherit (lib) mkDefault mkEnableOption mkIf mkPackageOption;
+
+  cfg = config.services.monado;
+
+in
+{
+  options.services.monado = {
+    enable = mkEnableOption "Monado wrapper and user service";
+
+    package = mkPackageOption pkgs "monado" { };
+
+    defaultRuntime = mkEnableOption "" // {
```

Nit: This is a pattern I recently discovered:

```suggestion
    defaultRuntime = mkEnableOption "" // mkOption {
```

(Preserves attribute checking.)